frameworks/phalcon

```diff
@@ -4,7 +4,7 @@ BUILD_DIR="$2"
 basedir="$( cd -P "$( dirname "$0" )" && pwd )"
 
 function requires_phalcon() {
-    jq --raw-output '.platform["ext-phalcon"]' < "$BUILD_DIR/composer.lock" | grep -q '^*$'
+    jq --raw-output '.platform' < "$BUILD_DIR/composer.lock" | grep -q 'ext-phalcon'
```

Comment:
It wasn't working?

Reply:
it wasn't working with Composer 2 because `platform` is now an array (and not anymore an object). This update is compatible with both Composer 1 and 2.

Reply:
Hmm great, thanks for the explanation